config.toml

```diff
@@ -91,6 +91,24 @@ pageRef = 'products'
 name = 'Product'
 weight = 6
 
+[[Languages.en.menu.main]]
+name = 'AI Assessments'
```

Comment:
When a user clicks on `Services` in the navigator bar, the dropdown menu will appear. There's is a link to the services page in the footer. Also, a user can navigate to the page by typing `/services` into their browser's address bar.